app/components/sessions/session-agenda-nav.hbs

```diff
@@ -1,25 +1,12 @@
-<ul class="auk-tabs auk-tabs--reversed" role="tablist">
-  <li class="auk-tabs__tab auk-tabs__hierarchical-back">
-    <LinkTo @route="agendas.overview">
-      <span class="auk-tabs__tab-inner">
-        <div class="auk-icon auk-icon--hierarchical-back auk-icon--default">
-          <i class="ki ki-hierarchical-back"></i>
-        </div>
-      </span>
-    </LinkTo>
-  </li>
-  <li class="auk-tabs__tab">
-    <LinkTo
-      @route="agenda.agendaitems.index"
-      @activeClass="active"
-      @query={{if this.currentAgendaItemId (hash anchor=this.currentAgendaItemId)}}
-    >
-      <span class="auk-tabs__tab-inner">
-        {{t "overview"}}
-      </span>
-    </LinkTo>
-  </li>
-  <li class="auk-tabs__tab">
+<Auk::Tabs @reversed={{true}}>
+  <Auk::Tab @route="agendas.overview" @isHierarchicalBack={{true}}/>
+  <Auk::Tab
+    @route="agenda.agendaitems.index"
+    @query={{if this.currentAgendaItemId (hash anchor=this.currentAgendaItemId)}}
+  >
+    {{t "overview"}}
+  </Auk::Tab>
+  <li role="tab" class="auk-tabs__tab">
     {{!-- TODO: get rid of async computed, so this wrapping if and the await can go --}}
     {{#if (await firstAgendaitemOfAgenda.id)}}
```

Comment:
@MikiDi @ValenberghsSven Enkel hier komt mijn kennis binnen Ember wat te kort vrees ik. Ik zou iets gelijkaardig aan `isActiveWithoutModels` (https://github.com/buschtoens/ember-link#isactivewithoutmodels) moeten gebruiken zodat de active styling correct op de tab komt te staan. `@current-when="agenda.agendaitems.agendaitem"` bij de `<Auk::Tab>` plaatsen lost het probleem alleszins niet op. Kan er iemand mij hierbij verder helpen?

Reply:
(Code staat tussen regel 25 - 38)

Reply:
ben aant kijken
in de oude linkTo stond `@activeClass="active"` om dat te bereiken precies